### PR TITLE
Change problematic verbiage of black/white – replace with block/allow

### PR DESCRIPTION
**Pull request by ali-sawyer**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
n/a

## Description
Replaces the offensive wording of "blacklist" and "whitelist" with the accepted industry standard terms of "blocklist" and "allowlist". As we are pushing this pack to more customers, it is important to update this verbiage to align with our values.

References:
- https://www.ciisec.org/wp-content/uploads/2023/10/DEI-Guide-to-Inclusive-Language-in-Cybersecurity.pdf
- https://www.splunk.com/en_us/blog/learn/blacklist-whitelist-inclusivity.html

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by rshunim** [comment]
Hi @ali-sawyer, Thanks for your fixes.
The PR is approved by me.
@tomer-pan I'd appreciate your review too.
Regarding the RN file . Please fix the file so the validation passes

**Review by rshunim — APPROVED** [review]

File: Packs/TIM_Processing/ReleaseNotes/1_1_27.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- Updated the terminology.
+##### TIM - Indicator Auto Processing
+
+- Updated the terminology.
+##### Renamed: TIM - Review Indicators Manually For Allowlisting
+
+- Updated terminology. Renamed from 'TIM - Review Indicators Manually For Whitelisting'.
+##### TIM - Review Indicators Manually
+
+- Updated the terminology.
```
Comment (ShirleyDenkberg) [inline Packs/TIM_Processing/ReleaseNotes/1_1_27.md:30]:
```suggestion
Updated the terminology. Renamed from 'Review Indicators Manually For Whitelisting'.

#### Layouts

##### Renamed: Review Indicators Manually For Allowlisting Incident

Updated the terminology. Renamed from 'Review Indicators Manually For Whitelisting Incident'.

#### Playbooks

##### TIM - Indicators Exclusion By Related Incidents

Updated the terminology.
##### TIM - Process Indicators - Fully Automated

Updated the terminology.
##### TIM - Indicator Auto Processing

Updated the terminology.
##### Renamed: TIM - Review Indicators Manually For Allowlisting

Updated terminology. Renamed from 'TIM - Review Indicators Manually For Whitelisting'.
##### TIM - Review Indicators Manually

Updated the terminology.
```

**Comment by ShirleyDenkberg** [comment]
@rshunim @tomer-pan Doc review completed.

File: Packs/TIM_Processing/IncidentTypes/incidenttype-Review_Indicators_Manually_For_Allowlisting.json [hunk 2]
```diff
@@ -1,15 +1,15 @@
 {
-	"id": "Review Indicators Manually For Whitelisting",
+	"id": "Review Indicators Manually For Allowlisting",
```
Comment (tomer-pan) [inline Packs/TIM_Processing/IncidentTypes/incidenttype-Review_Indicators_Manually_For_Allowlisting.json:2]: @ali-sawyer 
Changing incident type id and name will affect all other places using this object.
For example, in the playbook "TIM - Process AWS indicators" (FeedAWS pack) - there's a task named "Create Process Indicators Manually incident" which creates a new incident from that type. There are others as well.
If we change the id+name of this incident type, we should iterate through all of the other places which uses this incident type and change them as well to use the new name/id so they won't break.

**Comment by tomer-pan** [comment]
@ali-sawyer @rshunim
For all the places where the description is changed - that's not an issue.
BUT when changing IDs of objects, it will affect other objects using it.
For example other playbooks that count on the old name or even customers' custom playbooks or other objects which use the current layout or incident type IDs - we can potentially break those.
So, we should make sure to address all of the other OOTB content which uses the old IDs and maybe even show a breaking change note for this release which states that all custom content that relies on the layout or incident type might break and appropriate changes should be done to this custom content.

**Comment by ali-sawyer** [comment]
@tomer-pan that makes sense. I updated the references in the other feed packs that use the content that was renamed. Lmk if I should add a breaking change note.

**Comment by tomer-pan** [comment]
@ali-sawyer Thanks, now all looks good.
And yes, please add a breaking change note.

**Comment by rshunim** [comment]
@tomer-pan Please re-review the PR and approve if everything now is ok.
@ali-sawyer Please fix the validation error and make the build green.

**Comment by tomer-pan** [comment]
@ali-sawyer Regarding the BC notes for the other packs - I think it should also mention that ootb content should not break in case they will update the "TIM Processing" pack as well. It can only affect custom content which uses the relevant changed content.

**Comment by ali-sawyer** [comment]
@tomer-pan That makes sense. Updated.

**Review by tomer-pan — APPROVED** [review]

**Pull request merged**